octoprint_Spoolman/SpoolmanPlugin.py

```diff
@@ -175,3 +186,56 @@ def get_update_information(self):
                 ]
             }
         }
+
+    # Webhook for FilaMan to select a spool
+    @octoprint.plugin.BlueprintPlugin.route("/selectSpool", methods=["POST"])
+    def select_spool_webhook(self):
+        if not self._isInitialized:
+            return jsonify({"error": "Plugin not initialized"}), 503
+            
+        try:
+            data = request.json
+            
+            if not data:
+                return jsonify({"error": "Missing request data"}), 400
+                
+            spool_id = data.get("spool_id")
+            if spool_id is None:
+                return jsonify({"error": "Missing spool_id parameter"}), 400
+                
+            # Optional Parameter
+            tool = data.get("tool", "tool0")
+            
+            # Extrahiere nur die Nummer aus dem Tool-Namen
+            tool_index = tool.replace("tool", "") if tool.startswith("tool") else tool
+            
+            # Bestehende Settings holen
+            selected_spool_ids = self._settings.get([SettingsKeys.SELECTED_SPOOL_IDS])
+            
+            # Spule für das angegebene Tool auswählen
+            selected_spool_ids[tool_index] = {
+                'spoolId': str(spool_id)
+            }
+            
+            # Settings aktualisieren
+            self._settings.set([SettingsKeys.SELECTED_SPOOL_IDS], selected_spool_ids)
+            self._settings.save()
+            
+            # Event auslösen
+            self.triggerPluginEvent(
+                PluginEvents.SPOOL_SELECTED,
```

Comment:
Instead of using `PluginEvents` directly, you should be using `Events` from Octoprint itself (look for `from octoprint.events import Events` and other `triggerPluginEvent` calls). Octoprint ensures proper suffix is added.